packages/react/src/patterns/sub-patterns/ContentBlock/ContentBlock.js

```diff
@@ -136,6 +141,7 @@ function _renderCTA(cta) {
 }
 
 ContentBlock.propTypes = {
+  type: PropTypes.oneOf(['__inverse', '']),
```

Comment:
Minor thing, doesn't seem like the double underscore looks right here.

Reply:
oh yeah I meant to fix that